code/socialDistribution/utility.py

```diff
@@ -18,6 +18,10 @@ def make_request(method='GET', url='http://127.0.0.1:8000/', body='', headers=No
         r = requests.get(url)
     elif method == 'POST':
         r = requests.post(url, data=body, headers=headers)
+    elif method == 'DELETE':
+        r = requests.delete(url)
+    elif method == 'PUT':
+        r = requests.put(url, data=body)
```

Comment:
Sorry, this no longer exists in master. Can you move it to socialDistribution.requests.py and create a put and delete method?